packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -229,18 +233,37 @@ const sendAmountScreenOptions = ({
 }: {
   route: RouteProp<StackParamList, Screens.SendAmount>
 }) => {
+  const title = route.params?.isRequest
+    ? i18n.t('paymentRequestFlow:request')
+    : i18n.t('sendFlow7:send')
+
   return {
     ...emptyHeader,
     headerLeft: () => <BackButton eventName={CustomEventNames.send_amount_back} />,
-    headerTitle: () => (
-      <HeaderTitleWithBalance
-        title={i18n.t(`sendFlow7:${route.params?.isRequest ? 'request' : 'send'}`)}
-        token={CURRENCY_ENUM.DOLLAR}
-      />
-    ),
+    headerTitle: () => <HeaderTitleWithBalance title={title} token={CURRENCY_ENUM.DOLLAR} />,
   }
 }
 
+const validateRecipientIntroScreenOptions = () => ({
+  ...emptyHeader,
+  headerLeft: () => <CancelButton eventName={CustomEventNames.send_secure_cancel} />,
+})
+
+const validateRecipientAccountScreenOptions = () => ({
```

Comment:
nit: if these are only used by one screen it might be better to define them in its file as this file will become huge if every screen has a ScreenOptions defined here

Reply:
I think the issue is that there is no easy way to pass it into the `Navigator` file. Seems like defining them here is what we've been doing for now but can definitely see a point where it might make sense to break them out into their own file

Reply:
I dont follow that there is no easy way to pass it into the Navigator.

Reply:
I think I misunderstood. Do you mean just define the options on the component screen  but just export it to `Navigator`?

Reply:
like this 
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/backup/BackupPhrase.tsx#L49